pkg/apply/processor/install.go

```diff
@@ -167,6 +162,17 @@ func (c *InstallProcessor) PreProcess(cluster *v2.Cluster) error {
 	return nil
 }
 
+func replaceOrInsert(ss []string, s string) []string {
+	var ret []string
+	for i := range ss {
+		if ss[i] != s {
```

Comment:
This modification may change the order. Wouldn't it better to replace the content in ss directly?

if found: 
ss[i]==s 
   ss[i]=s
if not found:
   append(ss,s)